reframe/frontend/cli.py

```diff
@@ -903,9 +903,10 @@ def module_unuse(*paths):
         exec_policy.sched_flex_alloc_nodes = sched_flex_alloc_nodes
         parsed_job_options = []
         for opt in options.job_options:
+            optstr, *_ = re.split(r'=|\s+', opt)
```

Comment:
This is equivalent, I think:

```suggestion
            optstr = re.split(r'=|\s+', opt)[0]
```